pkg/volume/scaleio/sio_volume_test.go

```diff
@@ -425,54 +425,6 @@ func TestVolumeProvisionerWithIncompleteConfig(t *testing.T) {
 	}
 }
 
-func TestVolumeProvisionerWithZeroCapacity(t *testing.T) {
```

Comment:
Update this test to validate the minimum capacity is being used?

Reply:
fixed